api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -307,10 +309,12 @@ public void sendEgressRemediationEmail(DbUser dbUser, EgressRemediationAction ac
                 .put(EmailSubstitutionField.EGRESS_REMEDIATION_DESCRIPTION, remediationDescription)
                 .build());
 
-    sendWithRetries(
+    EgressAlertRemediationPolicy egressPolicy =
+        workbenchConfigProvider.get().egressAlertRemediationPolicy;
+    sendWithRetriesFrom(
```

Comment:
not a strong opinion but this can be an overload of `sendWithRetries()`

Reply:
I don't love the overload here because the current default behavior feels like a business logic decision, rather than a logical default - i.e. sending from support. Rereading this though, I'm wondering if it would be clearer to rename as follows:

```
sendWithRetriesFromSupport()
sendWithRetries()
```

thoughts?

Reply:
I started on this, then I realized the config value we're using as default is `config.mandrill.fromEmail`. The default is reasonable then... sorry - lets go with your overload suggestion.